### DISCUSSION TITLE
Support for Multiple Filters

### DISCUSSION THREAD

**Discussion by ManojKiranA** [filamentphp/filament#32] [Ideas]
For Example I have users table and i want to Apply Multiple Filtes simultaneously Such as i want to get `RD account type` with `account number between 1400-1500`. But Currently its not possible ATM.
![image](https://user-images.githubusercontent.com/30294553/109598168-c1877180-7b3e-11eb-8250-dc3ebd23b719.png)

It would be nice to have dedicated Filter Card

**Comment by danharrin** [comment]
Thanks for the suggestion! [It's on the roadmap](https://filamentadmin.com/docs/roadmap).

**Comment by ManojKiranA** [comment]
version [2.x](https://filamentadmin.com/docs/2.x/tables/filters) supports this
